source/BoardingPanel.cpp

```diff
@@ -333,81 +331,57 @@ bool BoardingPanel::KeyDown(SDL_Keycode key, Uint16 mod, const Command &command,
 				if(!yourCrew || !enemyCrew)
 					break;
 
+				double total = 0;
 				if(youAttack)
 				{
 					// Your chance of winning this round is equal to the ratio of
 					// your power to the enemy's power.
 					double yourAttackPower = attackOdds.AttackerPower(yourCrew);
 					double enemyDefensePower = attackOdds.DefenderPower(enemyCrew);
 					double total = yourAttackPower + enemyDefensePower;
-					if(total)
-					{
-						if(Random::Real() * total >= yourAttackPower)
-						{
-							++yourAttackCasualties;
-							you->AddCrew(-1);
-							if(!you->Crew())
-								break;
-						}
-						else
-						{
-							++victimDefenseCasualties;
-							victim->AddCrew(-1);
-							if(!victim->Crew())
-								break;
-						}
-					}
 				}
 				if(enemyAttacks)
 				{
 					double enemyAttackPower = defenseOdds.AttackerPower(enemyCrew);
 					double yourDefensePower = defenseOdds.DefenderPower(yourCrew);
 					double total = enemyAttackPower + yourDefensePower;
-					if(total)
+				}
+
+				if(total)
+				{
+					if(youAttack && Random::Real() * total >= yourAttackPower
```

Comment:
I'm not sure if this compiles, because yourAttackPower looks to be defined out of scope for this statement.

Reply:
Might be fixed in merge.